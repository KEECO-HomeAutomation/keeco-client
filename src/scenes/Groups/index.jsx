import React from 'react';

export const Groups = () => {
	return <p>Groups</p>;
};

export default Groups;
