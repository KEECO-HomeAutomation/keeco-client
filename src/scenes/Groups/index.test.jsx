import React from 'react';
import { mount } from 'enzyme';

import Groups from './index';

describe('<Groups />', () => {
	test('Should render without errors', () => {
		const comp = mount(<Groups />);
		expect(comp.length).toBe(1);
	});

	test('Should match snapshot', () => {
		expect(mount(<Groups />)).toMatchSnapshot();
	});
});
