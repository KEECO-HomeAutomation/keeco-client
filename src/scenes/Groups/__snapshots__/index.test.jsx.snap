// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Groups /> Should match snapshot 1`] = `
<Groups>
  <p>
    Groups
  </p>
</Groups>
`;
